pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -434,4 +434,7 @@ type DisruptionManagementSpec struct {
 	// it only works if managePodBudgetss is true.
 	// the default is 30 minutes
 	OSDMaintenenceTimeout time.Duration `json:"osdMaintenanceTimeout,omitempty"`
+
+	// This enables management of machinedisruptionbudgets
+	ManageMachineDisruptionBudgets bool `json:"manageMachineDisruptionBudgets,omitempty"`
```

Comment:
We'll want to move this under the [`disruptionManagement`](https://github.com/rook/rook/blob/87ca2086406674d64f6129d849bfa7764205f155/cluster/examples/kubernetes/ceph/cluster.yaml#L158)section when you rebase.